src/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ pkg_check_modules(GOBJECT REQUIRED gobject-2.0)
 pkg_check_modules(GIO REQUIRED gio-2.0)
 pkg_check_modules(GEE REQUIRED gee-0.8)
 pkg_check_modules(POPPLER REQUIRED poppler-glib)
-pkg_check_modules(GTK REQUIRED gtk+-3.0>=3.10)
+pkg_check_modules(GTK REQUIRED gtk+-3.0>=3.18)
```

Comment:
fullscreen_on_monitor was added with gtk 3.18, but I added a legacy version. cmake detects old versions and use just the fullscreen api.